composer.json

```diff
@@ -7,13 +7,19 @@
     "autoload-dev": {
         "psr-4": {
             "App\\Spec\\": "spec/"
-        }
+        },
+        "files": [
+          "src/prodecural.php"
+        ]
     },
     "config": {
         "bin-dir": "bin"
     },
     "minimum-stability": "dev",
     "prefer-stable": true,
+    "require": {
+      "php": ">=7.1"
```

Comment:
This is convention for describing the minim version of PHP required to run the package. For an example of this in the wild please see: https://packagist.org/packages/roots/bedrock